SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -113,20 +117,21 @@ public string GetCommandLineArgsLogText()
         }
 
         /// <summary>
-        /// Determines whether the text contains sensitive data that
-        /// should not be logged/written to file
+        /// Determines whether the text may contain sensitive data that should not be logged/written to file. It
+        /// looks at a blacklist of tokens in a case insensitive way.
         /// </summary>
         public static bool ContainsSensitiveData(string text)
         {
-            Debug.Assert(SensitivePropertyRegexps != null, "SensitivePropertyRegexps array should not be null");
-
-            if (text == null)
-            {
-                return false;
-            }
+            return ContainsSensitiveDataInternal(text, true);
```

Comment:
Well in fact all SonarQube properties are case sensitive: Why is there a need in the first place to do any case-insensitive comparison?